avm/utilities/e2e-template-assets/scripts/Copy-VhdToStorageAccount.ps1

```diff
@@ -62,17 +62,23 @@ begin {
     $currentVerbosePreference = $VerbosePreference
     $VerbosePreference = 'SilentlyContinue'
     $requiredModules = @(
-        'Az.ImageBuilder',
-        'Az.Storage'
+        @{ Name = 'Az.ImageBuilder'; Version = '0.4.0' },
```

Comment:
Can you pull latest main into your repository? I think the to file changes should vanish after